tests/nested/classic/hotplug/task.yaml

```diff
@@ -31,6 +31,7 @@ debug: |
     nested_exec "snap list"
     nested_exec "dmesg"
     nested_exec 'sudo jq -r ".data[\"hotplug-slots\"]" /var/lib/snapd/state.json'
+    set -e
```

Comment:
Does this have any effect if used at the end of section? AFAIU this will just appear at the end of spread-generated shell script. But maybe I'm misunderstanding the intent and how it spread works.

Reply:
I guess I don't remember exactly, but AIUI with spread is that all of the snippets are combined into a single shell script, including debug-each, prepare-each, etc. across all of the backend, project, suite, etc. so there could be snippets that are run after this section, and even if it's not, I just feel more comfortable resetting `-e` because shell scripts are so finnicky and treacherous in this regard I'd rather not risk it IMHO

Reply:
Ah, I missed +e a few lines above, resetting makes sense, yes. Thanks